services/galley/test/integration/API/Util.hs

```diff
@@ -130,6 +130,18 @@ getTeamMembersInternalTruncated tid n = do
       === statusCode
   responseJsonError r
 
+getLimitedTeamSize :: TeamId -> Int -> TestM LimitedTeamSize
+getLimitedTeamSize tid n = do
+  g <- view tsGalley
```

Comment:
ahh, way too many one-letter variables!  :)

Reply:
This was copy-pasta, sorry!

Reply:
The whole file is like that, I will leave it for now. I will tackle this next time I touch this file.